custom/theme.scss

```diff
@@ -1,6 +1,6 @@
-$light-background: #e9e5fb;
```

Comment:
Consider using consistent naming conventions for color variables to maintain a standardized style.